.travis.yml

```diff
@@ -1,6 +1,5 @@
 # vim: sw=2 ts=2 sts=2 expandtab
 
-dist: xenial
```

Comment:
Don't remove this line.
I don't want to remember which distro is the default.

Reply:
Done